FrontEnd/src/global.css

```diff
@@ -0,0 +1,44 @@
+ @import url('https://fonts.googleapis.com/css2?family=Inter&display=swap');
```

Comment:
I would recommend downloading this font and hosting it using the public/fonts folder to be less dependent on CDN

Reply:
fixed, thanks!